rapids-cmake/cpm/spdlog.cmake

```diff
@@ -29,10 +29,26 @@ across all RAPIDS projects.
 
 .. code-block:: cmake
 
-  rapids_cpm_spdlog( [BUILD_EXPORT_SET <export-name>]
+  rapids_cpm_spdlog( [FMT_OPTION <fmt-option-name>]
+                     [BUILD_EXPORT_SET <export-name>]
                      [INSTALL_EXPORT_SET <export-name>]
                      [<CPM_ARGS> ...])
 
+``FMT_OPTION``
+.. versionadded:: v23.04.00
+
+  Spdlog depends on the fmt library and offers multiple ways of handling this dependency when spdlog is built. This
+  option only controls the behavior when spdlog is fetched and built, NOT when an installed spdlog is found on the
+  system.
+
+  This option can be set to: `BUNDLED`, `EXTERNAL_FMT`, `EXTERNAL_FMT_HO`, or `STD_FORMAT`. If set to
+  `BUNDLED`, then spdlog will use its own bundled version of fmt. If set to `EXTERNAL_FMT` then spdlog will use the
+  `fmt::fmt` target and be linked with the fmt library. If set to `EXTERNAL_FMT_HO` then spdlog will use the
+  `fmt::fmt-header-only` target and be linked with a header only fmt library. If set to `STD_FORMAT` then spdlog
```

Comment:
“linked with a header only fmt library”

Is that the right wording? Can you link to a header-only library?

Reply:
From a CMake perspective you can. You use `target_link_libraries` on interface targets ( header only ) to get any include directories, defintions, etc that are need to use that library.

Reply:
In my brain I was thinking of the cmake `target_link_libraries` call which uses the `fmt::fmt-header-only` target. 🤷

Reply:
Thanks for the clarification!